README.md

```diff
@@ -602,9 +602,15 @@ The above will assign the values mapped in the dictionary as values to attribute
 * PostgreSQL server
 * Python >= 3.5 virtual environment
 
+**Fork the repository**
+```bash
+git clone https://github.com/your-user-name/django-enum-choices.git django-enum-choices-yourname
+cd pandas-yourname
```

Comment:
This should be `django-enum-choices-yourname`, right?

Reply:
haha, yes of course